infra/production/test-rollback-control-compatibility.sh

```
@@ -2,264 +2,18 @@
-[[ "$(<"$partial_state/previous-sha")" == "$APP_CURRENT" ]]
-[[ "$(<"$partial_state/previous-contract-sha")" == "$CONTROL_PREVIOUS" ]]
-[[ "$(<"$partial_state/active-mysql-volume")" == "$VOLUME" ]]
-
-printf 'OPS-027 rollback Control compatibility regression tests passed\n'
+SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" && pwd -P)"
+COMMON="$SCRIPT_DIR/release-common.sh"
+ROLLBACK="$SCRIPT_DIR/rollback.sh"
+DEPLOY="$SCRIPT_DIR/deploy.sh"
+
+bash -n "$COMMON" "$ROLLBACK" "$DEPLOY"
+! grep -Eq 'active-mysql-volume|PAWCYCLE_MYSQL_ENV_FILE|PAWCYCLE_MYSQL_VOLUME|MYSQL_DIGEST|compose (start|stop|rm).*mysql' "$COMMON" "$ROLLBACK" "$DEPLOY"
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)